### PR TITLE
typo fix in a string

### PR DESCRIPTION
**Pull request by stelt**

### CODE REVIEW

File: example/live_w_locator.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ $(function() {
                 $li.find("img").attr("src", result.frame);
                 $li.find("h4.code").html(result.codeResult.code + " (" + result.codeResult.format + ")");
                 $ul.prepend($li);
+                if(result.codeResult.code==="9789400506")location.href="https://medewerkers.saxion.nl/wps/myportal/!ut/p/b1/hY7LDoIwEEW_xS_olJZOWZpIoTyKFGKkG8PCGBIeG-P3WyNxYYLe3eSek7nEkY5CyAPJMWDkTNzcP4Zbfx-WuR9ftxOXUicxBJGgkFcS9AmVsUJQGaEHOg_ARvbw5bOCgc7aGgUqgISuvtG2bvI25XnchKAPpmD2kDCJb_9T89j6urW1LauMArDV__Hgz36TLtOVdB7DTezIyORGpaTePQFxT56z/dl4/d5/L2dBISEvZ0FBIS9nQSEh/";
```
Comment (serratus) [inline example/live_w_locator.js]: Please remove this line

File: example/live_w_locator.html [hunk 2]
```diff
@@ -34,10 +34,10 @@
                 <label>
                     <span>Barcode-Type</span>
                     <select name="decoder_readers">
-                        <option value="code_128" selected="selected">Code 128</option>
+                        <option value="code_128">Code 128</option>
                         <option value="code_39">Code 39</option>
                         <option value="code_39_vin">Code 39 VIN</option>
-                        <option value="ean">EAN</option>
+                        <option value="ean" selected="selected">EAN</option>
```
Comment (serratus) [inline example/live_w_locator.html]: Should not be the preferred option in the `master` branch. Please change that back

**Comment by serratus** [comment]
Thanks for adding the PR, but it seems like some files are specific to your needs. The only value I see is the fix of the typo. Anything else?

**Comment by stelt** [comment]
I meant to only suggest the typofix to you and not bother you with the rest, sorry that went wrong.
Before I change any code on my fork, I'll look into it, create a branch I think (if I have the terminology right).
I hoped to create a proof-of-concept : hold book for camera, based on specific ISBN go to specific URL (to show libraries are not 'book storage'; give related recommendations (video,dataset,PDF,audio,etc.)

**Comment by serratus** [comment]
Alright, thanks. I'll fix the typo and push it with the next release. Thanks for pointing that out. Are you OK when I close this PR without a merge then?

**Comment by stelt** [comment]
perfectly fine

**Pull request closed without merging**